### ISSUE TITLE
Non-work type fields in the data dictionary

### ISSUE THREAD

**Issue by awead** [psu-libraries/cho#559]
We should be able to define fields for file sets and files in the data dictionary. These would include terms for a file's original name, display name, use type (ex. thumbnail, original, service, etc.) as well as technical, rights, and administrative metadata.

Currently, our configuration schema assumes that core fields are included by default. However, for files and file sets, these terms may not apply. We should be able to define a group of arbitrary fields for any schema, preferably in config/schema_fields.yml.

These schemas could then control which fields are defined on non-work type resources such as file sets and files.

**Comment by awead** [comment]
Related to #512 and #550.

**Comment by ntallman** [comment]
Should this be closed or re-assigned? I think we decided not to add new fields to the data dictionary for files and file sets on 6/20, at least for now.

**Comment by awead** [comment]
Yes, let push this off until we need to administrative and technical metadata fields.

**Comment by awead** [comment]
Blocked by #66
